app/services/search_and_compare_api_service.rb

```diff
@@ -13,32 +13,32 @@ def api
   end
 
   class Request
-    class << self
-      # NOTE:
-      #   HTTP PUT "/api/courses" accepts a list of courses
-      #   It will only add or update, no DELETION.
-      def sync(course)
-        response = api.put(
-          "/api/courses/",
-          serialize(course)
-        )
+    attr_reader :response
 
-        response.success?
-      end
+    # NOTE:
+    #   HTTP PUT "/api/courses" accepts a list of courses
+    #   It will only add or update, no DELETION.
+    def sync(course)
```

Comment:
can you change that while you here to reflect its a list

Reply:
Yup, will change.